code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -107,10 +107,11 @@
 	icon_dead = "b_rabbit_dead"
 	icon_prefix = "b_rabbit"
 	minbodytemp = 0
-	eggsleft = 0
-	egg_type = null
 	speak = list()
 
+/mob/living/simple_animal/chicken/rabbit/normal/EggLayingComponent()
```

Comment:
this should be moved with all the other rabbits

Reply:
in fact, this is for winter cabin and so should really get a special path name + name + desc for its ability to survive cold temperatures

Reply:
i removed the subtype, it wasn't used in the end